docs/installation/additional-settings-for-developers/index.md

```diff
@@ -45,3 +45,39 @@ export ROS_DOMAIN_ID=X
 ```
 
 For more information, see [here](https://docs.ros.org/en/foxy/Concepts/About-Domain-ID.html#the-ros-domain-id).
+
+### Tuning DDS
+
+Autoware uses DDS for inter-node communication. [ROS 2 documentation](https://docs.ros.org/en/humble/How-To-Guides/DDS-tuning.html) recommends users to tune DDS to utilize its capability. Especially, receive buffer size is the critical parameter for Autoware. If the parameter is not large enough, Autoware will failed in receiving large data like point cloud or image.
+
+For example, to execute Autoware with CycloneDDS, prepare config file. A sample config file, named as `cyclonedds_config.xml`, is given below.
+
+```xml
+<?xml version="1.0" encoding="UTF-8" ?>
+<CycloneDDS xmlns="https://cdds.io/config" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://cdds.io/config
+https://raw.githubusercontent.com/eclipse-cyclonedds/cyclonedds/master/etc/cyclonedds.xsd">
+    <Domain id="any">
+        <Internal>
+            <SocketReceiveBufferSize min="10MB"/>
+        </Internal>
+    </Domain>
+</CycloneDDS>
+```
+
+Set the config file path and enlarge the Linux kernel maximum buffer size before launching Autoware.
+
+```bash
+export CYCLONEDDS_URI=file:///absolute/path/to/cyclonedds_config.xml
+sudo sysctl -w net.core.rmem_max=2147483647
+```
+
+Refer to [ROS 2 documentation](https://docs.ros.org/en/humble/How-To-Guides/DDS-tuning.html) for more information. Reading user guide for chosen DDS is helpful for more understanding.
+
+### Tuning DDS for multiple host computers (for advanced users)
+
+When Autoware runs on multiple host computers, IP Fragmentation should be taken account of. As [ROS 2 documentation](https://docs.ros.org/en/humble/How-To-Guides/DDS-tuning.html#cross-vendor-tuning) recommends, parameters for IP Fragmentation should be given as well as the following example.
```

Comment:
```suggestion
When Autoware runs on multiple host computers, IP Fragmentation should be taken into account. As [ROS 2 documentation](https://docs.ros.org/en/humble/How-To-Guides/DDS-tuning.html#cross-vendor-tuning) recommends, parameters for IP Fragmentation should be set as shown in the following example.
```
nit: grammar

Reply:
@xmfcx  Thank you, and I do fix the document as you suggested.